sdk/solana/tbrv3/network/network.json

```diff
@@ -1,4 +1,4 @@
 {
   "mainnet": "AtTrxsPbTfBhC9uwwJGJbkFMux78t5EWTAXAbwUW8yC7",
```

Comment:
Let's set mainnet to the same key just in case.

Reply:
Agreed, i'll make the change and save the encrypted key for reuse.